### ISSUE TITLE
WinRM Initialization Error

### ISSUE THREAD

**Issue by KyhleOhlinger** [Orange-Cyberdefense/GOAD#17]
### OS And Relevant Info
**Description:**	Ubuntu 22.04 LTS
**Release:**	22.04
**Codename:**	jammy

**Vagrant:** 2.2.19
**Ansible:** Core 2.12.6

### Issue
Running Vagrant Up results in WinrRM Error:

_Edited: Included additional output_
```
(.venv) test@AD-Lab:~/Desktop/GOAD$ vagrant up --provision
Bringing machine 'DC01' up with 'virtualbox' provider...
==> DC01: Importing base box 'StefanScherer/windows_2019'...
==> DC01: Matching MAC address for NAT networking...
==> DC01: Checking if box 'StefanScherer/windows_2019' version '2021.05.15' is up to date...
==> DC01: Setting the name of the VM: GOAD_DC01_1657733508313_12037
==> DC01: Clearing any previously set network interfaces...
==> DC01: Preparing network interfaces based on configuration...
    DC01: Adapter 1: nat
    DC01: Adapter 2: hostonly
==> DC01: Forwarding ports...
    DC01: 3389 (guest) => 3389 (host) (adapter 1)
    DC01: 5985 (guest) => 55985 (host) (adapter 1)
    DC01: 5986 (guest) => 55986 (host) (adapter 1)
    DC01: 22 (guest) => 2222 (host) (adapter 1)
==> DC01: Running 'pre-boot' VM customizations...
==> DC01: Booting VM...
==> DC01: Waiting for machine to boot. This may take a few minutes...
    DC01: WinRM address: 127.0.0.1:55985
    DC01: WinRM username: vagrant
    DC01: WinRM execution_time_limit: PT2H
    DC01: WinRM transport: plaintext
An error occurred executing a remote WinRM command.

Shell: Cmd
Command: hostname
Message: Digest initialization failed: initialization error

```

### Debug Output

[debug_log.txt](https://github.com/Orange-Cyberdefense/GOAD/files/9104455/debug_log.txt)

**Comment by KyhleOhlinger** [comment]
I did wait for the VM to boot completely and reran the provisioner, but it still threw the same error. 

Additionally, it doesn't seem as if the IP is being set correctly for the machines. DC01 is meant to have .10 but the VM has .5 - could be because initialization hasn't been completed though.

**Comment by 6b6f6c6a61** [comment]
You might want to try version 2.2.18 of Vagrant as suggested by [PaQoo](https://github.com/PaQoo) in issue #12. This fixed these errors for me on Ubuntu 22.04 LTS. However I still have problems with Ansible afterwards as described in the issue.
